shardingsphere-governance/shardingsphere-governance-core/src/test/java/org/apache/shardingsphere/governance/core/registry/process/subscriber/ProcessRegistrySubscriberTest.java

```diff
@@ -0,0 +1,137 @@
+ */
+
+package org.apache.shardingsphere.governance.core.registry.process.subscriber;
+
+import java.util.Collection;
+import java.util.Collections;
+import org.apache.shardingsphere.governance.core.registry.process.event.ExecuteProcessReportEvent;
+import org.apache.shardingsphere.governance.core.registry.process.event.ExecuteProcessSummaryReportEvent;
+import org.apache.shardingsphere.governance.core.registry.process.event.ExecuteProcessUnitReportEvent;
+import org.apache.shardingsphere.governance.core.registry.process.event.ShowProcessListRequestEvent;
+import org.apache.shardingsphere.governance.repository.spi.RegistryCenterRepository;
+import org.apache.shardingsphere.infra.executor.sql.context.ExecutionUnit;
+import org.apache.shardingsphere.infra.executor.sql.process.model.ExecuteProcessConstants;
+import org.apache.shardingsphere.infra.executor.sql.process.model.ExecuteProcessContext;
+import org.apache.shardingsphere.infra.executor.sql.process.model.ExecuteProcessUnit;
+import org.apache.shardingsphere.infra.executor.sql.process.model.yaml.YamlExecuteProcessContext;
+import org.apache.shardingsphere.infra.executor.sql.process.model.yaml.YamlExecuteProcessUnit;
+import org.apache.shardingsphere.infra.yaml.engine.YamlEngine;
+import org.junit.Before;
+import org.junit.Test;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import org.mockito.MockitoAnnotations;
+
+public final class ProcessRegistrySubscriberTest {
+
+    @Mock
+    private RegistryCenterRepository repository;
+
+    @InjectMocks
+    private ProcessRegistrySubscriber processRegistrySubscriber;
+
+    @Before
+    public void init() {
+        MockitoAnnotations.initMocks(this);
+    }
+
+    @Test
+    public void testLoadShowProcessListData() {
+        ShowProcessListRequestEvent showProcessListRequestEvent = mock(ShowProcessListRequestEvent.class);
+        Mockito.when(repository.getChildrenKeys(any())).thenReturn(Collections.singletonList("abc"));
+        Mockito.when(repository.get(any())).thenReturn("abc");
+        processRegistrySubscriber.loadShowProcessListData(showProcessListRequestEvent);
+        Mockito.verify(repository, times(1)).get(any());
+    }
+
+    @Test
+    public void testReportExecuteProcessSummary() {
+        ExecuteProcessContext executeProcessContext = mock(ExecuteProcessContext.class);
+        ExecuteProcessSummaryReportEvent event = mock(ExecuteProcessSummaryReportEvent.class);
+        Mockito.when(event.getExecuteProcessContext()).thenReturn(executeProcessContext);
+        Mockito.when(executeProcessContext.getExecutionID()).thenReturn("id");
+        processRegistrySubscriber.reportExecuteProcessSummary(event);
+        Mockito.verify(event, times(1)).getExecuteProcessContext();
```

Comment:
Could you verify `persist` method is invoked with certain `key` parameter, e.g. `/executionnodes/id`? Just like `RegistryCenterTest`.